docker-compose.yaml

```diff
@@ -0,0 +1,60 @@
+version: "3.5"
```

Comment:
This file should not be on the root of the repo, as it is github-related only.
Copy it to some other location (`.github/workflows/assets/docker-compose.yaml` maybe?) and use it in the github actions either directly (`docker-compose -f `) or copy before use.